models/public/ssh-mxnet/model.yml

```diff
@@ -32,4 +32,5 @@ model_optimizer_args:
   - --mean_values=data[123.68,116.779,103.939]
   - --reverse_input_channels
 framework: mxnet
+quantizable: yes
```

Comment:
Given that `ssh-mxnet` is (at least temporarily) unavailable, I suggest you revert this for now, so that the PR can be properly tested. If it comes back, you can make it quantizable in a separate PR.